clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/GCEUtil.groovy

```diff
@@ -46,6 +48,24 @@ class GCEUtil {
 
   public static final String TARGET_POOL_NAME_PREFIX = "tp"
 
+  private static final int TIMEOUT_RETRY_COUNT = 10
+
+  // Will attempt to call operation once, if it throws a SocketTimeoutException it will retry at most TIMEOUT_RETRY_COUNT times using the
+  // retry closure.
+  static Operation protectAgainstTimeout(Closure<Operation> operation, Closure<Operation> retry, Logger log = null, String operationName = '') {
+    (TIMEOUT_RETRY_COUNT + 1).times { i ->
```

Comment:
This isn't being used yet, but I suspect we will want to plug this in soon after trying out the new timeout.